test/special-cases/edge-cases-expected-fails.yaml

```diff
@@ -313,3 +313,49 @@ spec:
       - video=true
     jobTemplate: "apiVersion: batch/v1\nkind: Job\nspec:\n  template:\n    spec:\n      containers:\n        - name: \"{{ .Name }}\"\n          image: {{ .Image }}\n          resources:\n            limits:\n              memory: 30Mi\n              cpu: 1\n"
     activeDeadlineSeconds: 600
+---
+apiVersion: tests.testkube.io/v3
```

Comment:
Shouldn't it have `negativeTest: true`?

Reply:
> No - we have separate ones. There are negative tests (with `-negative` names), and there are separate `expected-fail` tests to also have actual failures (that's needed to test different edge-cases).

Reply:
no - there are negative tests (with `-negative` names), and there are separate `expected-fail` tests to also have actual failures (that's needed to test different edge-cases)